src/index.js

```diff
@@ -14,7 +14,11 @@ let tryRequire = (pathOrId: string | number) => {
   try {
     // $FlowIgnore
     return babelInterop(require(pathOrId));
-  } catch (err) {}
+  } catch (error) {
+    if (error.name === "SyntaxError") {
```

Comment:
Not entirely sure I follow. Do you mean adding more cases or changing how we check what kind of error it is?